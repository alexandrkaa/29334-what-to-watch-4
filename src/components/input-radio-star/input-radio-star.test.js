import React from 'react';
import renderer from 'react-test-renderer';
import InputRadioStar from './input-radio-star.jsx';

const idx = 1;
const onRadioChange = jest.fn();
describe(`<InputRadioStar /> should match snapshot`, () => {
  it(`<InputRadioStar /> should match snapshot (enabled)`, () => {
    const postCommentInProgress = false;
    const tree = renderer
      .create(
          <InputRadioStar
            isDisabled={postCommentInProgress}
            key={idx}
            id={idx + 1}
            isChecked={false}
            onRadioChange={onRadioChange}
          />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`<InputRadioStar /> should match snapshot (disabled)`, () => {
    const postCommentInProgress = true;
    const tree = renderer
      .create(
          <InputRadioStar
            isDisabled={postCommentInProgress}
            key={idx}
            id={idx + 1}
            isChecked={false}
            onRadioChange={onRadioChange}
          />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
