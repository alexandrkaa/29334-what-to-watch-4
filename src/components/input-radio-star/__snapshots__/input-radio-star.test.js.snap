// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputRadioStar /> should match snapshot <InputRadioStar /> should match snapshot (disabled) 1`] = `
Array [
  <input
    checked={false}
    className="rating__input"
    disabled={true}
    id="star-2"
    name="rating"
    onChange={[Function]}
    type="radio"
    value={2}
  />,
  <label
    className="rating__label"
    htmlFor="star-2"
  >
    Rating 2
  </label>,
]
`;

exports[`<InputRadioStar /> should match snapshot <InputRadioStar /> should match snapshot (enabled) 1`] = `
Array [
  <input
    checked={false}
    className="rating__input"
    disabled={false}
    id="star-2"
    name="rating"
    onChange={[Function]}
    type="radio"
    value={2}
  />,
  <label
    className="rating__label"
    htmlFor="star-2"
  >
    Rating 2
  </label>,
]
`;
